src/docs/database-migrations.mdx

```diff
@@ -88,6 +88,16 @@ class MyMigrationTest(TestMigrations):
 
 To run the test locally, run `pytest` with `--migrations` flag. For example, `pytest -v --migrations tests/getsentry/migrations/test_0XXX_migration_name.py`.
 
+### Backup Testing
+When you add or change a model, an error message in CI may appear explaining that one or multiple tests "produced an `export.json` backup file that was missing the above models".
+In order to resolve this, there are two steps:
+1. Add the new or modified model to the exhaustive organization in [testutils/helpers/backups.py](https://github.com/getsentry/sentry/blob/master/src/sentry/testutils/helpers/backups.py#L366) by creating an instance of your model, for example by invoking MyModel.objects.create(). This ensures the presence of the new model when creating the snapshot and during testing.
```

Comment:
All the other links to Sentry's GH files in the docs point to the file within specific commit, maybe here that would also be good idea, because the line number can easily change in the future